src/widget/wlibrarytableview.cpp

```diff
@@ -42,7 +42,7 @@ WLibraryTableView::WLibraryTableView(QWidget* parent,
     setVerticalScrollBarPolicy(Qt::ScrollBarAlwaysOn);
     setAlternatingRowColors(true);
 
-    loadVScrollBarPosState();
+    //loadVScrollBarPosState();
```

Comment:
Don't comment out code. Either delete or keep it. The next one reading this code will not know why it has been commented out and why it is still here.